src/Quasar/Mount/Type.purs

```diff
@@ -49,5 +49,5 @@ fromName = case _ of
   "spark-hdfs" → SparkHDFS
   "spark-ftp" → SparkFTP
   "spark-local" → SparkLocal
-  "mimir" → Mimir
+  "local" → Local
```

Comment:
If we're still making the changes in here, this value on the left should remain as `"mimir"`, since it's the name of the mount type in Quasar.